lib/ui/widgets/common/custom_drawer/custom_drawer.dart

```diff
@@ -96,14 +96,13 @@ class CustomDrawer extends StackedView<CustomDrawerModel> {
                 height: 24.h,
                 ),
                 SizedBox(width: 16.w,),
-                Text("DARK MODE",
+                Text(S.current.dark_mode,
                 style: typography?.titleBold12?.copyWith(
                   color: palette?.gray11
                 ),),
                 SizedBox(width: 98.w,),
-                Switch(value: true, onChanged: (value) {
-                  
-                },)
+                Switch(value: true, onChanged: viewModel.toggleSwitch,
```

Comment:
shouldnt this be `Switch(value: viewModel.isSwitched, onChanged: viewModel.toggleSwitch,` by now? You created a state variable for it already.

Reply:
oh! thanks